packages/devtools_app/lib/src/error_badge_manager.dart

```diff
@@ -23,6 +28,12 @@ class ErrorBadgeManager extends DisposableController
     NetworkScreen.id: ValueNotifier<int>(0),
     LoggingScreen.id: ValueNotifier<int>(0),
   };
+  final _activeErrors = <String,
+      ValueNotifier<LinkedHashMap<InspectorInstanceRef, DevToolsError>>>{
```

Comment:
we have the problem here that InspectorInstanceRef is specific to the inspector, so this `_activeErrors` won't apply to other pages, like the logging view or performance view for example. It may take some extra work to generalize this, so I'm fine with you changing this to `_activeInspectorErrors` of type `ValueNotifier<LinkedHashMap<InspectorInstanceRef, DevToolsError>>` for now. Can you also add a todo explaining that we should generalize this to be compatible with errors for other screens?

Reply:
How about we change the key back from `InspectorInstanceRef` to just `String`? I think the rest of the code is fairly generic, it's just this `key` that is a wrapper over the string.

Reply:
sgtm

Reply:
Done! (b2c3652e)